docs/sources/trackers/google-tag-manager/snowplow-template/tag-types/index.md

```diff
@@ -0,0 +1,327 @@
+
+
+### Error Tracking
+
+The Errors tracker tag provides a way to manually track any errors that occur on your website.
+
+<details plain>
+<summary>Error Tracking Parameters</summary>
+
+| **Name**   | **Required?** | **Description**                     | **Example**              |
+| ---------- | ------------- | ----------------------------------- | ------------------------ |
+| `message`  | Yes           | Error message                       | 'Cannot get user object' |
+| `filename` | No            | Filename or URL                     | 'shop.js'                |
+| `lineno`   | No            | Line number of problem code chunk   | 23                       |
+| `colno`    | No            | Column number of problem code chunk | 45                       |
+| `error`    | No            | JS `ErrorEvent`                     | {Stack trace string}     |
+
+</details>
+
+### Page View
+
+You can provide a Custom Page Title if you wish, and you can add a [custom context](/docs/sources/trackers/snowplow-tracker-protocol/custom-events/context-entities/index.md) to the request, as also described [here](/docs/sources/trackers/javascript-trackers/web-tracker/tracking-events/page-views/index.md#tracking-a-page-view).
+
+If you enable Page Activity Tracking, page pings will be sent at intervals that you can specify.
+
+The Callback Function is something you can set to a JavaScript function. If you set the callback, then instead of sending the page ping to Snowplow, the function gets invoked instead.
+
+
+### Link Click Tracking
+
+The Track Link Click event is similar to regular parameter-based events, as it lets you add parameters and track a link click as a manually encoded hit.
+
+The Enable Automatic Link Click Tracking adds listeners to the page, which will track clicks on links permitting they adhere to the blacklisted/whitelisted class names you can optionally provide. It is intended to trigger [early on a page](https://support.google.com/tagmanager/answer/7679319) to begin listening for click events, _not_ to trigger on click events like [Click Triggers](https://support.google.com/tagmanager/answer/7679320).
+
+The Fix Middle-click Tracking adds a fix for some browsers where middle-clicks were not tracked properly.
+
+If you check Track HTML Content Of Clicked Link, then the full text content of the link element will be sent to Snowplow as well.
+
+:::note
+
+This setting enables the tracker's native [Link Click Tracking plugin](/docs/sources/trackers/javascript-trackers/web-tracker/tracking-events/link-click/index.md), it does not directly fire an event.
+
+:::
+
+### Form Tracking
+
+Form tracking has just two options, because filters and transformations won’t work with the custom template (due to lack of support for processing HTML elements). The options are to set form and/or field blacklists and whitelists.
```

Comment:
are they really still called black/white lists or allow/deny lists?

Reply:
They are in fact allow/deny now, good catch!